protocols/migration/migration_reports_4_0_0_to_reports_5_0_0.py

```diff
@@ -0,0 +1,573 @@
+        return new_assembly
+
+    def migrate_called_genotype_to_variant_call(self, old_instance):
+        """
+        NOTE: fields that cannot be filled "vaf"
+        :type old_instance: reports_4_0_0.CalledGenotype
+        :rtype reports_5_0_0.VariantCall
+        :return:
+        """
+        new_instance = self.convert_class(
+            self.new_model.VariantCall, old_instance)  # :type: reports_5_0_0.VariantCall
+
+        # rename gelId to participantId
+        new_instance.participantId = old_instance.gelId
+
+        # rename genotype to zygosity
+        new_instance.zygosity = old_instance.genotype
+
+        # sets allele origin to germline, we fail to set maternal/paternal origin or de novo status
+        new_instance.alleleOrigins = [reports_5_0_0.AlleleOrigin.germline_variant]
+
+        # NOTE: fields that are lost: copyNumber
+        # NOTE: fields that cannot be filled: vaf, alleleOrigins
+
+        return self.validate_object(
+            object_to_validate=new_instance, object_type=self.new_model.VariantCall
+        )
+
+    def migrate_report_event(self, old_instance):
+        """
+
+        :type old_instance: reports_4_0_0.ReportEvent
+        :rtype reports_5_0_0.ReportEvent
+        :return:
+        """
+        new_instance = self.convert_class(self.new_model.ReportEvent, old_instance)
+
+        # phenotypes has been changed to a list
+        new_instance.phenotypes = [old_instance.phenotype]
+
+        # panelName and panelVersion are now inside a dedicated object
+        if old_instance.panelName is not None:
+            new_instance.genePanel = self.new_model.GenePanel(
+                panelName=old_instance.panelName)
+            if old_instance.panelVersion is not None:
+                new_instance.genePanel.panelVersion = old_instance.panelVersion
+
+        # genomic feature has been changed to a list
+        new_instance.genomicEntities = [self.migrate_genomic_feature(old_instance.genomicFeature)]
+
+        # variant classification is now in a complex object
+        if old_instance.variantClassification is not None:
+            map_variant_classification = {
+                reports_4_0_0.VariantClassification.benign_variant: reports_5_0_0.ClinicalSignificance.benign,
+                reports_4_0_0.VariantClassification.likely_benign_variant: reports_5_0_0.ClinicalSignificance.likely_benign,
+                reports_4_0_0.VariantClassification.variant_of_unknown_clinical_significance:
+                    reports_5_0_0.ClinicalSignificance.VUS,
+                reports_4_0_0.VariantClassification.likely_pathogenic_variant:
+                    reports_5_0_0.ClinicalSignificance.likely_pathogenic,
+                reports_4_0_0.VariantClassification.pathogenic_variant: reports_5_0_0.ClinicalSignificance.pathogenic,
+                reports_4_0_0.VariantClassification.not_assessed: None
+            }
```

Comment:
just to save the key:value pairs being over multiple lines could we utilise some renaming, maybe something like:
```
old_variant_classification = reports_4_0_0.VariantClassification
new_clinical_significance = reports_5_0_0.ClinicalSignificance
map_variant_classification = {
    old_variant_classification.benign_variant: new_clinical_significance.benign,
    old_variant_classification.likely_benign_variant: new_clinical_significance.likely_benign,
    old_variant_classification.variant_of_unknown_clinical_significance: new_clinical_significance.VUS,
    old_variant_classification.likely_pathogenic_variant: new_clinical_significance.likely_pathogenic,
    old_variant_classification.pathogenic_variant: new_clinical_significance.pathogenic,
    old_variant_classification.not_assessed: None
}
```